LoopKitUI/Extensions/Environment+ColorPalette.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Environment+ColorPalette.swift
+//  LoopKitUI
+//
+//  Created by Nathaniel Hamming on 2020-07-29.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import SwiftUI
+
+private struct ColorPaletteKey: EnvironmentKey {
+    static var defaultValue: ColorPalette? = nil
```

Comment:
I think we should make this non-optional, even if the default colors are bogus, for the sake of avoiding optionality where we don't really expect it